client/src/components/App.js

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect } from "react";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import { hot } from "react-hot-loader/root";
-
+import three from "../../public/images/DALL·E 2023-05-03 12.11.41 - Three astronauts  watching a movie on a screen on the moon pixel art.png"
```

Comment:
Im going to allow this, but ideally this image is hosted, and then you just put the image url in here